cmd/dfget/app/root.go

```diff
@@ -23,118 +23,128 @@ var (
 	filter     string
 )
 
+var cfg = config.NewConfig()
+
 var rootCmd = &cobra.Command{
 	Use:   "dfget",
 	Short: "The dfget is the client of Dragonfly.",
 	Long:  "The dfget is the client of Dragonfly, a non-interactive P2P downloader.",
 	Run: func(cmd *cobra.Command, args []string) {
-		initConfig(args)
-		cfg.Ctx.ClientLogger.Infof("cmd params:%q", args)
-		err := core.Start(cfg.Ctx)
-		util.Printer.Println(resultMsg(cfg.Ctx, time.Now(), err))
-		if err != nil {
-			os.Exit(err.Code)
-		}
-		os.Exit(0)
+		runDfget(args)
 	},
 }
 
 func init() {
 	initFlags()
 }
 
+// runDfget do some init operation.
+func runDfget(args []string) {
+	// initialize logger and get properties
```

Comment:
If you want to add comments for your functions, please obey this rule:

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences